Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/database/data/PointerDBAdapterV3.java

```diff
@@ -0,0 +1,124 @@
+/* ###
+ * IP: GHIDRA
+ * REVIEWED: YES
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ghidra.program.database.data;
+
+import ghidra.util.exception.VersionException;
+
+import java.io.IOException;
+
+import db.*;
+
+class PointerDBAdapterV3 extends PointerDBAdapter {
+	final static int VERSION = 3;
+
+	private Table table;
+
+	PointerDBAdapterV3(DBHandle handle, boolean create) throws VersionException, IOException {
+
+		if (create) {
+			table = handle.createTable(POINTER_TABLE_NAME, SCHEMA, new int[] { PTR_CATEGORY_COL });
+		}
+		else {
+			table = handle.getTable(POINTER_TABLE_NAME);
+			if (table == null) {
+				throw new VersionException("Missing Table: " + POINTER_TABLE_NAME);
+			}
+			else if (table.getSchema().getVersion() != VERSION) {
+				int version = table.getSchema().getVersion();
+				if (version < VERSION) {
+					throw new VersionException(true);
+				}
+				throw new VersionException(VersionException.NEWER_VERSION, false);
+			}
+		}
+	}
+
+	/* (non-Javadoc)
+	 * @see ghidra.program.database.data.PointerDBAdapter#createRecord(long, long, int)
+	 */
```

Comment:
You may remove these javadocs on overrides when they don't contribute anything.

Reply:
Ok

Reply:
Yes they are all over the place but aren't used anymore so they can be removed.

Reply:
OK, gonna remove it